### PR TITLE
Missing API docs for Application

### PR DESCRIPTION
**Pull request by mairaw**
/cc @carlossanlop @RussKie

Second commit shows my edits

### CODE REVIEW

File: xml/System.Windows.Forms/Application.xml [hunk 1]
```diff
@@ -1975,8 +1976,8 @@ static class Program
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value that indicates whether visual styles are enabled for the application.</summary>
+        <value><see langword="true" /> if visual styles are enabled; otherwise, <see langword="false" />.</value>
         <remarks>To be added.</remarks>
```
Comment (mairaw) [inline xml/System.Windows.Forms/Application.xml]: @RussKie should we say something here about when a mode wouldn't be set? Can we list the reasons?

File: xml/System.Windows.Forms/Application.xml [hunk 2]
```diff
@@ -718,8 +718,8 @@
         <ReturnType>System.Windows.Forms.HighDpiMode</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the current high DPI mode for the process.</summary>
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/Application.xml]: process, or application?

Comment (RussKie) [reply xml/System.Windows.Forms/Application.xml]: I think it should be "application"

File: xml/System.Windows.Forms/Application.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.Windows.Forms/Application.xml]: @mairaw feel free to edit my suggestion:

```suggestion
        <remarks>

## Remarks

The visual styles can be enabled by calling <xref:System.Windows.Forms.Application.EnableVisualStyles>.

The visual styles will not be enabled if the OS does not support them (which is unlikely), or theming is disabled at the OS level.


        </remarks>
```

Comment (dreddy-work) [reply xml/System.Windows.Forms/Application.xml]: Many times our classic framework applications were targeted to windows 7  in the manifest file. VisualStyles are 'by default' enabled on windows 8 and above. For core, we do not have manifest.  So, need to check how this works/useful.

Comment (dreddy-work) [reply xml/System.Windows.Forms/Application.xml]: @Tanya-Solyanik

Comment (Tanya-Solyanik) [reply xml/System.Windows.Forms/Application.xml]: Some server SKUs  don't support visual styles, so I would remove `(which is unlikely)` clause.

File: xml/System.Windows.Forms/Application.xml (hunk 2 above)
Comment (mairaw) [reply xml/System.Windows.Forms/Application.xml]:
```suggestion
        <summary>Gets the current high DPI mode for the application.</summary>
```

File: xml/System.Windows.Forms/Application.xml (hunk 1 above)
Comment (Tanya-Solyanik) [reply xml/System.Windows.Forms/Application.xml]: EnableVisualStyles and manifest file works fine in Core applicatins, here is a core app with disabled visual styles:
image

**Pull request merged**